src/million/createElement.ts

```diff
@@ -32,7 +32,13 @@ export const createElement = (vnode?: VNode | null, attachField = true): DOMNode
         } else if (propName.charCodeAt(5) === COLON_CHAR) {
           el.setAttributeNS(XLINK_NS, propName, String(propValue));
         }
-      } else if (el[propName] !== undefined && !(el instanceof SVGElement)) {
```

Comment:
There is a bit more elegant solution. Because both list and form are `null` value instead of `undefined`, you can check the `null` case like this instead of hard code:

```js
el[propName] !== undefined && 
el[propName] !== null && 
!(el instanceof SVGElement) &&
propName in el
```

Reply:
Could you also make this change in `useProps.ts`?

Reply:
ok~